django_project/lesson/views/worksheet.py

```diff
@@ -564,3 +570,23 @@ def get_context_data(pk):
         'attachment; filename={}-worksheet {}.zip'.format(
             project.name, downloaded_module)
     return zip_response
+
+
+class WorksheetSampleData(View):
+    """Download sample data."""
+
+    def get(self, request, *args, **kwargs):
+        numbering = self.request.GET.get('q', '')
```

Comment:
can you get this version number from the project model? so you don't need to pass this variable to view because it seems redundant.

Reply:
The numbering is not saved in the model instance.
It's coming from dynamic forloop in the template. 
https://github.com/kartoza/prj.app/blob/58aec98309b3aa1ca18b9c776e5b9bac2a086126/django_project/lesson/templates/section/includes/row-worksheet.html#L12
and the numbering might be different between author and normal user, since the author have view in draft (unpublished) worksheet, but the normal users don't.

Reply:
LGTM